src/olympia/addons/cron.py

```diff
@@ -8,7 +8,7 @@
 from django.utils.encoding import force_text
 
 import multidb
-from celery.task.sets import TaskSet
+from celery import group
```

Comment:
is `group` a drop-in replacement for `TaskSet`?

Reply:
Correct, it is.